Frends.SMTP.SendEmail/Frends.Smtp.SendEmail/SendEmailTask.cs

```diff
@@ -9,143 +9,153 @@
+		/// [Documentation](https://github.com/FrendsPlatform/Frends.SMTP/tree/main/Frends.SMTP.SendEmail)
+		/// </summary>
+		/// <returns>
+		/// Object { bool EmailSent, string StatusString }
+		/// </returns>
+		public static Output SendEmail([PropertyTab] Input message, [PropertyTab] Attachment[] attachments, [PropertyTab] Options SMTPSettings, CancellationToken cancellationToken)
+		{
+			var output = new Output();
+
+			using (var client = InitializeSmtpClient(SMTPSettings))
+			{
+				using (var mail = InitializeMailMessage(message))
+				{
+					if (attachments != null)
+						foreach (var attachment in attachments)
+						{
+							if (attachment.AttachmentType == AttachmentType.FileAttachment)
+							{
+								ICollection<string> allAttachmentFilePaths = GetAttachmentFiles(attachment.FilePath);
+
+								if (attachment.ThrowExceptionIfAttachmentNotFound && allAttachmentFilePaths.Count == 0)
+									throw new FileNotFoundException(string.Format("The given filepath \"{0}\" had no matching files", attachment.FilePath), attachment.FilePath);
+
+								if (allAttachmentFilePaths.Count == 0 && !attachment.SendIfNoAttachmentsFound)
+								{
+									output.StatusString = string.Format("No attachments found matching path \"{0}\". No email sent.", attachment.FilePath);
+									output.EmailSent = false;
+									return output;
+								}
+
+								foreach (var fp in allAttachmentFilePaths)
+								{
+									mail.Attachments.Add(new System.Net.Mail.Attachment(fp));
+								}
+							}
+
+							if (attachment.AttachmentType == AttachmentType.AttachmentFromString
+								&& !string.IsNullOrEmpty(attachment.stringAttachment.FileContent))
+							{
+								mail.Attachments.Add(System.Net.Mail.Attachment.CreateAttachmentFromString
+									(attachment.stringAttachment.FileContent, attachment.stringAttachment.FileName));
+							}
+
+						}
+
+					cancellationToken.ThrowIfCancellationRequested();
+
+					client.Send(mail);
+
+					output.EmailSent = true;
+					output.StatusString = string.Format("Email sent to: {0}", mail.To.ToString());
+
+					return output;
+				}
+			}
+		}
+
+		/// <summary>
+		/// Initializes new SmtpClient with given parameters.
+		/// </summary>
+		private static SmtpClient InitializeSmtpClient(Options settings)
+		{
+			var smtpClient = new SmtpClient
+			{
+				Port = settings.Port,
+				DeliveryMethod = SmtpDeliveryMethod.Network,
+				UseDefaultCredentials = settings.UseWindowsAuthentication,
+				EnableSsl = settings.UseSsl,
+				Host = settings.SMTPServer
+			};
+
+			if (!settings.UseWindowsAuthentication && !string.IsNullOrEmpty(settings.UserName))
+				smtpClient.Credentials = new NetworkCredential(settings.UserName, settings.Password);
+
+			return smtpClient;
+		}
+
+		/// <summary>
+		/// Initializes new MailMessage with given parameters. Uses default value 'true' for IsBodyHtml
+		/// </summary>
+		public static MailMessage InitializeMailMessage(Input input)
```

Comment:
Why is this changed from private to public?

Reply:
Changed back to private.